packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/add-oidc-provider/template.html

```diff
@@ -0,0 +1,47 @@
+<form name="addOidcProviderForm" id="addOidcProvider" data-ng-submit="$ctrl.onAddOidcProviderConfirm()">
+    <oui-modal
+        data-on-dismiss="$ctrl.goBack()"
+        data-primary-label="{{:: 'pci_projects_project_kubernetes_details_service_add_oidc_provider_action_add' | translate}}"
+        data-primary-disabled="addOidcProviderForm.$invalid"
+        data-secondary-action="$ctrl.onAddOidcProviderCancel()"
+        data-secondary-label="{{:: 'pci_projects_project_kubernetes_details_service_add_oidc_provider_action_cancel' | translate}}"
+        data-loading="$ctrl.isAdding"
+        data-heading="{{:: 'pci_projects_project_kubernetes_details_service_add_oidc_provider_title' | translate}}"
+    >
+        <p
+            data-translate="pci_projects_project_kubernetes_details_service_add_oidc_provider_description"
+        ></p>
+
+        <!--OIDC provider URL-->
+        <oui-field
+            data-label="{{::'pci_projects_project_kubernetes_details_service_add_oidc_provider_field_url' | translate}}"
+        >
+            <span data-translate="pci_projects_project_kubernetes_details_service_add_oidc_provider_field_url_extra"></span>
+            <input
+                type="text"
+                class="oui-input"
+                data-ng-init="labelKeyId = 'oidcProviderModelUrl'"
+                data-ng-attr-name="{{labelKeyId}}"
+                data-ng-attr-id="{{labelKeyId}}"
+                data-ng-model="$ctrl.oidcProviderModel.issuerUrl"
+                required
+            />
+        </oui-field>
+
+        <!--OIDC provider clientId-->
+        <oui-field
+            data-label="{{::'pci_projects_project_kubernetes_details_service_add_oidc_provider_field_client_id' | translate}}"
+        >
+            <span data-translate="pci_projects_project_kubernetes_details_service_add_oidc_provider_field_client_id_extra"></span>
+            <input
+                type="text"
+                class="oui-input"
+                data-ng-init="labelKeyId = 'oidcProviderModelClientId'"
```

Comment:
prefer to have all initialisation done in controller, easy to refactor

Reply:
Same as here:
https://github.com/ovh/manager/pull/5860#discussion_r744556096